Basic project setup

olexandrd opened the pull request.

kazamov reviewed (commented):
1. Please add a `.vscode/settings.json` file with content 
`{
	"editor.formatOnSave": true,
	"[css]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode"
	},
	"[html]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode"
	}
}`
Thus prettier will be applied automatically on file save for all workspace users.

2. Install packages locally and commit package-lock.json file. Thus everybody will have the same versions of packages.

olexandrd commented:
> 1. Please add a `.vscode/settings.json` file with content
>        `{ "editor.formatOnSave": true, "[css]": { "editor.defaultFormatter": "esbenp.prettier-vscode" }, "[html]": { "editor.defaultFormatter": "esbenp.prettier-vscode" } }`
>        Thus prettier will be applied automatically on file save for all workspace users.
> 
> 2. Install packages locally and commit package-lock.json file. Thus everybody will have the same versions of packages.
> 
> 3. Add .gitignore file.
> 
> 4. Change `html` folder name to `partials` as it is described in the [Instruction](https://www.edu.goit.global/uk/learn/13571785/20712526/20713447/homework)

Thanks for valuable review! 
1. Added.
2. Commited `package-lock.json`
3. Already present on the init commit.
4. Changed
